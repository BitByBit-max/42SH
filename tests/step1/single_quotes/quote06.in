echo -E 'hello hi how jhdhirherp\n'
