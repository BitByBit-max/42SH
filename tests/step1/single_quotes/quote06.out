hello hi how jhdhirherp\n
